valhalla/baldr/graph_utils.h

```diff
@@ -6,14 +6,14 @@
 
```

Comment:
this whole file seems misplaced. could we just move the functions unto an existing utils header?

Reply:
I didn’t feel it’s fitting anywhere.. def not mjolnir, and it’s not really midgard material.

Reply:
why dont we just delete this file entirely and then we can change the interface of elevationbuilder::build to take an unordered_set and make a dequeue out of it and check the filesystem etc if that is what it needs. just move the functionality into the main function, or, if it means testing is less verbose thats ok with me. rather have less random files hanging around tbh